### DISCUSSION TITLE
Lidarr - is there a folder I can drop in manually downloaded music and have beets process it then add it to Lidarr?

### DISCUSSION THREAD

**Discussion by t0fr** [RandomNinjaAtk/arr-scripts#277] [General]
Title. If I download music manually elsewhere for higher release quality, or other music I've had on the side, is there a folder I can just drop it into such that beets picks it up and adds it to the beets/lidarr library?

**Comment by jaycedk** [comment]
Why not just manually import them into lidarr ?

**Reply by t0fr** [reply]
Well, I'd like for beets to process them so that the album art, lyrics, and replaygain can all be done.

**Reply by jaycedk** [reply]
TBH this a helper script for Lidarr.
I would guess its out of the scope of this.
But a PR could help.

**Comment by Treavous** [comment]
This is a feature I would like too. But my scope is more along the lines of having it scan the already imported in Lidarr stuff. I had a lot of stuff downloaded already before I knew about the scripts
